test/amount-test.js

```diff
@@ -1182,11 +1187,11 @@ describe('Amount', function() {
 
   describe('amount limits', function() {
     it ('max JSON wire limite', function() {
-      assert.strictEqual(Amount.bi_xns_max.toString(), '9000000000000000000');
+      assert.strictEqual(Amount.bi_xns_max.toString(), '100000000000000000');
```

Comment:
There's thing like old offers, with takerPays of some huge amount. Recall that offers need only be partially funded.